### ISSUE TITLE
[HOLD on #53650] [$250] Android 3-button navigation bar overlaps the "Get Stated" button of the "Welcome to Expensify!" modal

### ISSUE THREAD

**Issue by m-natarajan** [Expensify/App#53779]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:**
**Reproducible in staging?:** Needs Reproduction
**Reproducible in production?:** Needs Reproduction
**If this was caught on HybridApp, is this reproducible on New Expensify Standalone?:**
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):** Issue reported on device : Oscal Tiger 12
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** @rayane-djouah 
**Slack conversation** (hyperlinked to channel name): [expensify_bugs](https://expensify.slack.com/archives/C049HHMV9SM/p1733518294921299)

## Action Performed:
1. Make sure that your Android device uses 3-button navigation bar
2. Create fresh account
3. Finish or skip onboarding flow.
4. Open HybridApp (OldDot) and go to NewDot.
5. After opening NewDot one should see Explanation modal.
## Expected Result:
Android 3-button navigation bar doesn't overlap the "Get Stated" button
## Actual Result:
Android 3-button navigation bar overlaps the "Get Stated" button
## Workaround:
Unknown
## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [x] Android: Standalone
- [x] Android: HybridApp
- [ ] Android: mWeb Chrome
- [ ] iOS: Standalone
- [ ] iOS: HybridApp
- [ ] iOS: mWeb Safari
- [ ] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

<details>
  <summary>Add any screenshot/video evidence</summary>

  
</details>

<img src="https://github.com/user-attachments/assets/ca4a8b40-e4de-4f71-9b7f-4afbc1e3a3bd" width="40%" height="40%">

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~021868824322335113052</li>
        <li>Upwork Job ID: 1868824322335113052</li>
        <li>Last Price Increase: 2024-12-17</li>
    </ul>
</details>

<details><summary>Issue Owner</summary>Current Issue Owner: @Pujan92</details>

**Comment by MelvinBot** [comment]
This has been labelled "Needs Reproduction". Follow the steps here: https://stackoverflowteams.com/c/expensify/questions/16989

**Comment by RachCHopkins** [comment]
Yes, reproed! Though this does seem to be an issue everywhere, like https://github.com/Expensify/App/issues/53650 has the same issue with a different button.

**Comment by RachCHopkins** [comment]
Oops, didn't meant to sit on it!

**Comment by RachCHopkins** [comment]
Sorry @Pujan92 I just heard back on https://github.com/Expensify/App/issues/53650 - we're going to fix it there I believe, so I'm just going to pop this on hold until that's done.

**Comment by FitseTLT** [comment]
@RachCHopkins This is issue is a Hybrid one the modal is only shown for Hybrid App and I also suggest a retest because from the code I can see that the safe area padding is correctly being applied. It might be already fixed. 👍

**Comment by Pujan92** [comment]
I see a similar issue here https://github.com/Expensify/App/issues/53842 and it may require a generic solution and all should be handled in the same PR.

**Comment by RachCHopkins** [comment]
https://github.com/Expensify/App/issues/53650 is still a work in progress

**Comment by RachCHopkins** [comment]
Updated priority to reflect that we're waiting on the above.

**Comment by RachCHopkins** [comment]
Ready for a retest.

**Comment by izarutskaya** [comment]
Issue is reproducible on build  9.1.9-1

https://github.com/user-attachments/assets/cede8c95-142a-4408-9f90-c19582e35e77

**Comment by RachCHopkins** [comment]
Thanks! I thought this would be fixed!
